.pubnub.yml

```diff
@@ -30,7 +37,7 @@ changelog:
       -
         text: "Fixed HereNow single channel response decoding"
         type: bug
-    date: 2019-XX-XX
+    date: 2019-10-08
```

Comment:
Is the suggestion to always update the release with the latest commit date or do we want to update this date before/after release with the actual date of release?

Reply:
Good question, not sure we have an exact process. I was just attempting to do it while I was getting ready to release.